package.json

```diff
@@ -22,7 +22,10 @@
     "build:sample-modules": "node scripts/build-sample-modules.js",
     "start:watch": "node scripts/watch-server.js",
     "prestart:prod-sample": "npm run build:prod-sample",
+    "prestart:prod-sample:debug": "npm run build:prod-sample",
     "start:prod-sample": "docker-compose -f ./prod-sample/docker-compose.yml up --abort-on-container-exit",
+    "start:prod-sample:debug": "docker-compose -f ./prod-sample/docker-compose.yml -f ./prod-sample/docker-compose.debug.yml up --abort-on-container-exit",
```

Comment:
This is a really cool option to start one app in prod mode, do we need to document it anywhere?